routes/web.php

```diff
@@ -13,6 +18,23 @@
 |
 */
 
-Route::get('/', function () {
-    return view('welcome');
-});
+Route::get('/products', [ProductController::class, 'index']);
```

Comment:
Try to use resource routes

Reply:
i don't use all crud operations related to Product in ProductController, some of them i use in AdministrationProductController

Reply:
Here I mean the whole file and all your routes. see where it can be used and make sure you use it.

Reply:
Okey